Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/README.md

```diff
@@ -34,6 +34,7 @@ In order to use the CrowdStrike Falcon integration, your API client must be prov
 - Alerts - Read and Write
 - Hosts - Read
 - IOC Manager - Read and Write
+- On-Demand Scans (ODS) - Read and Write
```

Comment:
add to Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon_description.md

Reply:
done